 Use `pm2 monit` to monitor CPU and Memory usage er-backend
root@DELRHDERS001P:~/do# ^C
root@DELRHDERS001P:~/do# pm2 logsa
[PM2][ERROR] Command not found

usage: pm2 [options] <command>

pm2 -h, --help             all available commands and options
pm2 examples               display pm2 usage examples
pm2 <command> -h           help on a specific command

Access pm2 files in ~/.pm2
root@DELRHDERS001P:~/do# pm2 logs
[TAILING] Tailing last 15 lines for [all] processes (change the value with --lines option)
/root/.pm2/pm2.log last 15 lines:
PM2        | 2025-07-23T13:50:08: PM2 log: App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2        | 2025-07-23T13:50:08: PM2 log: App [er-backend:0] starting in -fork mode-
PM2        | 2025-07-23T13:50:08: PM2 log: App [er-backend:0] online
PM2        | 2025-07-23T13:50:09: PM2 log: App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2        | 2025-07-23T13:50:09: PM2 log: App [er-backend:0] starting in -fork mode-
PM2        | 2025-07-23T13:50:09: PM2 log: App [er-backend:0] online
PM2        | 2025-07-23T13:50:10: PM2 log: App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2        | 2025-07-23T13:50:10: PM2 log: App [er-backend:0] starting in -fork mode-
PM2        | 2025-07-23T13:50:10: PM2 log: App [er-backend:0] online
PM2        | 2025-07-23T13:50:10: PM2 log: App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2        | 2025-07-23T13:50:10: PM2 log: App [er-backend:0] starting in -fork mode-
PM2        | 2025-07-23T13:50:10: PM2 log: App [er-backend:0] online
PM2        | 2025-07-23T13:50:11: PM2 log: App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2        | 2025-07-23T13:50:11: PM2 log: App [er-backend:0] starting in -fork mode-
PM2        | 2025-07-23T13:50:11: PM2 log: App [er-backend:0] online

/root/.pm2/logs/er-backend-out.log last 15 lines:
/root/.pm2/logs/er-backend-error.log last 15 lines:
0|er-backe |     at async ScramSHA256.auth (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:37:20)
0|er-backe |     at async performInitialHandshake (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:104:13)
0|er-backe |     at async connect (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:24:9) {
0|er-backe |   errorLabelSet: Set(2) { 'HandshakeError', 'ResetPool' },
0|er-backe |   errorResponse: {
0|er-backe |     ok: 0,
0|er-backe |     errmsg: 'Authentication failed.',
0|er-backe |     code: 18,
0|er-backe |     codeName: 'AuthenticationFailed'
0|er-backe |   },
0|er-backe |   ok: 0,
0|er-backe |   code: 18,
0|er-backe |   codeName: 'AuthenticationFailed',
0|er-backe |   connectionGeneration: 0
0|er-backe | }

0|er-backend  | (node:1973902) [MONGODB DRIVER] Warning: useNewUrlParser is a deprecated option: useNewUrlParser has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | (Use `node --trace-warnings ...` to show where the warning was created)
0|er-backend  | (node:1973902) [MONGODB DRIVER] Warning: useUnifiedTopology is a deprecated option: useUnifiedTopology has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | MongoDB connection error: MongoServerError: Authentication failed.
0|er-backend  |     at Connection.sendCommand (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:305:27)
0|er-backend  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
0|er-backend  |     at async Connection.command (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:333:26)
0|er-backend  |     at async continueScramConversation (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:131:15)
0|er-backend  |     at async ScramSHA256.auth (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:37:20)
0|er-backend  |     at async performInitialHandshake (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:104:13)
0|er-backend  |     at async connect (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:24:9) {
0|er-backend  |   errorLabelSet: Set(2) { 'HandshakeError', 'ResetPool' },
0|er-backend  |   errorResponse: {
0|er-backend  |     ok: 0,
0|er-backend  |     errmsg: 'Authentication failed.',
0|er-backend  |     code: 18,
0|er-backend  |     codeName: 'AuthenticationFailed'
0|er-backend  |   },
0|er-backend  |   ok: 0,
0|er-backend  |   code: 18,
0|er-backend  |   codeName: 'AuthenticationFailed',
0|er-backend  |   connectionGeneration: 0
0|er-backend  | }
PM2           | App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2           | App [er-backend:0] starting in -fork mode-
PM2           | App [er-backend:0] online
0|er-backend  | (node:1973926) [MONGODB DRIVER] Warning: useNewUrlParser is a deprecated option: useNewUrlParser has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | (Use `node --trace-warnings ...` to show where the warning was created)
0|er-backend  | (node:1973926) [MONGODB DRIVER] Warning: useUnifiedTopology is a deprecated option: useUnifiedTopology has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | MongoDB connection error: MongoServerError: Authentication failed.
0|er-backend  |     at Connection.sendCommand (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:305:27)
0|er-backend  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
0|er-backend  |     at async Connection.command (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:333:26)
0|er-backend  |     at async continueScramConversation (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:131:15)
0|er-backend  |     at async ScramSHA256.auth (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:37:20)
0|er-backend  |     at async performInitialHandshake (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:104:13)
0|er-backend  |     at async connect (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:24:9) {
0|er-backend  |   errorLabelSet: Set(2) { 'HandshakeError', 'ResetPool' },
0|er-backend  |   errorResponse: {
0|er-backend  |     ok: 0,
0|er-backend  |     errmsg: 'Authentication failed.',
0|er-backend  |     code: 18,
0|er-backend  |     codeName: 'AuthenticationFailed'
0|er-backend  |   },
0|er-backend  |   ok: 0,
0|er-backend  |   code: 18,
0|er-backend  |   codeName: 'AuthenticationFailed',
0|er-backend  |   connectionGeneration: 0
0|er-backend  | }
PM2           | App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2           | App [er-backend:0] starting in -fork mode-
PM2           | App [er-backend:0] online
0|er-backend  | (node:1973939) [MONGODB DRIVER] Warning: useNewUrlParser is a deprecated option: useNewUrlParser has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | (Use `node --trace-warnings ...` to show where the warning was created)
0|er-backend  | (node:1973939) [MONGODB DRIVER] Warning: useUnifiedTopology is a deprecated option: useUnifiedTopology has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | MongoDB connection error: MongoServerError: Authentication failed.
0|er-backend  |     at Connection.sendCommand (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:305:27)
0|er-backend  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
0|er-backend  |     at async Connection.command (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:333:26)
0|er-backend  |     at async continueScramConversation (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:131:15)
0|er-backend  |     at async ScramSHA256.auth (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:37:20)
0|er-backend  |     at async performInitialHandshake (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:104:13)
0|er-backend  |     at async connect (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:24:9) {
0|er-backend  |   errorLabelSet: Set(2) { 'HandshakeError', 'ResetPool' },
0|er-backend  |   errorResponse: {
0|er-backend  |     ok: 0,
0|er-backend  |     errmsg: 'Authentication failed.',
0|er-backend  |     code: 18,
0|er-backend  |     codeName: 'AuthenticationFailed'
0|er-backend  |   },
0|er-backend  |   ok: 0,
0|er-backend  |   code: 18,
0|er-backend  |   codeName: 'AuthenticationFailed',
0|er-backend  |   connectionGeneration: 0
0|er-backend  | }
PM2           | App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2           | App [er-backend:0] starting in -fork mode-
PM2           | App [er-backend:0] online
0|er-backend  | (node:1973952) [MONGODB DRIVER] Warning: useNewUrlParser is a deprecated option: useNewUrlParser has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | (Use `node --trace-warnings ...` to show where the warning was created)
0|er-backend  | (node:1973952) [MONGODB DRIVER] Warning: useUnifiedTopology is a deprecated option: useUnifiedTopology has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | MongoDB connection error: MongoServerError: Authentication failed.
0|er-backend  |     at Connection.sendCommand (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:305:27)
0|er-backend  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
0|er-backend  |     at async Connection.command (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:333:26)
0|er-backend  |     at async continueScramConversation (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:131:15)
0|er-backend  |     at async ScramSHA256.auth (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:37:20)
0|er-backend  |     at async performInitialHandshake (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:104:13)
0|er-backend  |     at async connect (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:24:9) {
0|er-backend  |   errorLabelSet: Set(2) { 'HandshakeError', 'ResetPool' },
0|er-backend  |   errorResponse: {
0|er-backend  |     ok: 0,
0|er-backend  |     errmsg: 'Authentication failed.',
0|er-backend  |     code: 18,
0|er-backend  |     codeName: 'AuthenticationFailed'
0|er-backend  |   },
0|er-backend  |   ok: 0,
0|er-backend  |   code: 18,
0|er-backend  |   codeName: 'AuthenticationFailed',
0|er-backend  |   connectionGeneration: 0
0|er-backend  | }
PM2           | App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2           | App [er-backend:0] starting in -fork mode-
PM2           | App [er-backend:0] online
0|er-backend  | (node:1973965) [MONGODB DRIVER] Warning: useNewUrlParser is a deprecated option: useNewUrlParser has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | (Use `node --trace-warnings ...` to show where the warning was created)
0|er-backend  | (node:1973965) [MONGODB DRIVER] Warning: useUnifiedTopology is a deprecated option: useUnifiedTopology has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | MongoDB connection error: MongoServerError: Authentication failed.
0|er-backend  |     at Connection.sendCommand (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:305:27)
0|er-backend  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
0|er-backend  |     at async Connection.command (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:333:26)
0|er-backend  |     at async continueScramConversation (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:131:15)
0|er-backend  |     at async ScramSHA256.auth (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:37:20)
0|er-backend  |     at async performInitialHandshake (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:104:13)
0|er-backend  |     at async connect (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:24:9) {
0|er-backend  |   errorLabelSet: Set(2) { 'HandshakeError', 'ResetPool' },
0|er-backend  |   errorResponse: {
0|er-backend  |     ok: 0,
0|er-backend  |     errmsg: 'Authentication failed.',
0|er-backend  |     code: 18,
0|er-backend  |     codeName: 'AuthenticationFailed'
0|er-backend  |   },
0|er-backend  |   ok: 0,
0|er-backend  |   code: 18,
0|er-backend  |   codeName: 'AuthenticationFailed',
0|er-backend  |   connectionGeneration: 0
0|er-backend  | }
PM2           | App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2           | App [er-backend:0] starting in -fork mode-
PM2           | App [er-backend:0] online
0|er-backend  | (node:1973978) [MONGODB DRIVER] Warning: useNewUrlParser is a deprecated option: useNewUrlParser has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | (Use `node --trace-warnings ...` to show where the warning was created)
0|er-backend  | (node:1973978) [MONGODB DRIVER] Warning: useUnifiedTopology is a deprecated option: useUnifiedTopology has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | MongoDB connection error: MongoServerError: Authentication failed.
0|er-backend  |     at Connection.sendCommand (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:305:27)
0|er-backend  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
0|er-backend  |     at async Connection.command (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:333:26)
0|er-backend  |     at async continueScramConversation (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:131:15)
0|er-backend  |     at async ScramSHA256.auth (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:37:20)
0|er-backend  |     at async performInitialHandshake (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:104:13)
0|er-backend  |     at async connect (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:24:9) {
0|er-backend  |   errorLabelSet: Set(2) { 'HandshakeError', 'ResetPool' },
0|er-backend  |   errorResponse: {
0|er-backend  |     ok: 0,
0|er-backend  |     errmsg: 'Authentication failed.',
0|er-backend  |     code: 18,
0|er-backend  |     codeName: 'AuthenticationFailed'
0|er-backend  |   },
0|er-backend  |   ok: 0,
0|er-backend  |   code: 18,
0|er-backend  |   codeName: 'AuthenticationFailed',
0|er-backend  |   connectionGeneration: 0
0|er-backend  | }
PM2           | App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2           | App [er-backend:0] starting in -fork mode-
PM2           | App [er-backend:0] online
0|er-backend  | (node:1973991) [MONGODB DRIVER] Warning: useNewUrlParser is a deprecated option: useNewUrlParser has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | (Use `node --trace-warnings ...` to show where the warning was created)
0|er-backend  | (node:1973991) [MONGODB DRIVER] Warning: useUnifiedTopology is a deprecated option: useUnifiedTopology has no effect since Node.js Driver version 4.0.0 and will be removed in the next major version
0|er-backend  | MongoDB connection error: MongoServerError: Authentication failed.
0|er-backend  |     at Connection.sendCommand (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:305:27)
0|er-backend  |     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
0|er-backend  |     at async Connection.command (/opt/erapp/server/node_modules/mongodb/lib/cmap/connection.js:333:26)
0|er-backend  |     at async continueScramConversation (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:131:15)
0|er-backend  |     at async ScramSHA256.auth (/opt/erapp/server/node_modules/mongodb/lib/cmap/auth/scram.js:37:20)
0|er-backend  |     at async performInitialHandshake (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:104:13)
0|er-backend  |     at async connect (/opt/erapp/server/node_modules/mongodb/lib/cmap/connect.js:24:9) {
0|er-backend  |   errorLabelSet: Set(2) { 'HandshakeError', 'ResetPool' },
0|er-backend  |   errorResponse: {
0|er-backend  |     ok: 0,
0|er-backend  |     errmsg: 'Authentication failed.',
0|er-backend  |     code: 18,
0|er-backend  |     codeName: 'AuthenticationFailed'
0|er-backend  |   },
0|er-backend  |   ok: 0,
0|er-backend  |   code: 18,
0|er-backend  |   codeName: 'AuthenticationFailed',
0|er-backend  |   connectionGeneration: 0
0|er-backend  | }
PM2           | App [er-backend:0] exited with code [1] via signal [SIGINT]
PM2           | App [er-backend:0] starting in -fork mode-
PM2           | App [er-backend:0] online

